### ISSUE TITLE
Change to Imperial units

### ISSUE THREAD

**Issue by wbste** [joaovitoriasilva/endurain#57]
Hey, thanks all for the great app. Working well!

I dug through the docs and code, but couldn't find it...is there an option to change the units to imperial?

**Comment by joaovitoriasilva** [comment]
Hi, thanks for reaching out and for the feedback.

Currently there is no option to change the units.

**Comment by TheBig-O** [comment]
Stupid Americans don't understand the metric system. I can't tell if my pace is good or bad. 😝

**Comment by TheBig-O** [comment]
That's a shame. Truly enjoy the app. Imperial units would be such a great addition.

**Comment by joaovitoriasilva** [comment]
Hello, I am starting to implement this. See screenshot bellow
![Screenshot 2025-01-13 at 16 32 32](https://github.com/user-attachments/assets/44ad2384-1996-407f-a597-b7323f44671f)

**Comment by joaovitoriasilva** [comment]
Hello again. This is implemented in dev branch and will be merged for next release.

![Image](https://github.com/user-attachments/assets/215eb8b7-e362-4fbd-a5d1-1f446e7d8033)

**Comment by joaovitoriasilva** [comment]
Hello again. I just released dev version "dev_05022025" with imperial units support. Can you please test and let me know if everything is okay?
You can change user units settings on user settings.

Please backup your DB before applying the update. I don't expect any issue, however...

Thank you.

**Comment by joaovitoriasilva** [comment]
Implemented on v0.8.0

**Comment by TheBig-O** [comment]
I'm sure I'm doing something wrong, but my instance is not updating to v0.8.0. I've deleted the stack and recreated it but it still says v0.6.5
As a result, I see no way to change to imperial units? I'm using the `latest` tag. should I be using a different branch?
`Image: ghcr.io/joaovitoriasilva/endurain/frontend:latest`

**Comment by joaovitoriasilva** [comment]
@TheBig-O I received an email with an update message from you to this issue, but now I can't see it.
Did you deleted the message? Is it solved? Do you still need help?

Thank you

**Comment by TheBig-O** [comment]
Yes, apparently I missed a change in the Docker-Compose. That was the problem. It wouldn't update to the current version without the updated compose. 
Thanks for checking though! GREAT SUPPORT!!!
